docs/src/developer/install.md

```diff
@@ -1,36 +1,118 @@
-# Installing miniSASS Django-CMS project
+# Setting up a Local Development Environment
 
-This is currently a basic Django-CMS site. No extra Django applications have been added yet. 
-To install the pre-requisites needed for this site to run, do the following:
+## Prerequisites
 
-## Docker Deployments
+- Any Linux distribution for the operating system
+- Docker installed on the local machine
```

Comment:
We also need docker-compose. We can also give links i.e https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04 or https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04